owl_msgs/action/ArmMoveJointsGuarded.action

```diff
@@ -0,0 +1,15 @@
+# goal definition
+bool relative
+float64[] angles
```

Comment:
To be ROS-like and future-proof, you could call this "positions." And it appears the word "angle" is propagated through the code.

Reply:
Good to know. I am unsure whether a change was requested here?

Reply:
It's up to you. Right now we're dealing with only angles. If we ever added a different type of joint that didn't rotate, then some refactoring would be in order for the code to make sense.

Reply:
I do like positions better than angles, but I will opt to stick with the unification doc. It's a good note for the future.

Reply:
I agree it was best to stick with the unified doc on the name.  Good thing to point out as a comment in that doc.